Generated React Frontend module no longer builds due to renamed adobe npm modules

## Expected Behaviour
building latest archetype should work.

### Actual Behaviour
build currently fails while testing generating a project with a react frontend module:
```
[INFO] [INFO] --- frontend-maven-plugin:1.7.6:npm (npm install) @ testing-frontend-react.ui.frontend ---
[INFO] [INFO] Running 'npm install' in /home/circleci/repo/target/test-classes/projects/frontend-react/project/testing-frontend-react/ui.frontend
[INFO] [WARNING] npm WARN deprecated @adobe/cq-spa-component-mapping@1.1.1: Package has been renamed. Please use @adobe/aem-spa-component-mapping
[INFO] [WARNING] npm WARN deprecated @adobe/cq-spa-page-model-manager@1.1.5: Package has been renamed. Please use @adobe/aem-spa-page-model-manager
[INFO] [WARNING] npm WARN deprecated @adobe/cq-react-editable-components@1.3.2: Package has been renamed. Please use @adobe/aem-react-editable-components
...
[INFO] [ERROR] internal/modules/cjs/loader.js:582
[INFO] [ERROR]     throw err;
[INFO] [ERROR]     ^
[INFO] [ERROR] 
[INFO] [ERROR] Error: Cannot find module '/home/circleci/repo/target/test-classes/projects/frontend-react/project/testing-frontend-react/ui.frontend/node_modules/@adobe/cq-react-editable-components/deprecated.js'
[INFO] [ERROR]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
[INFO] [ERROR]     at Function.Module._load (internal/modules/cjs/loader.js:506:25)
[INFO] [ERROR]     at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
[INFO] [ERROR]     at startup (internal/bootstrap/node.js:285:19)
[INFO] [ERROR]     at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)
...
[INFO] [ERROR] 
[INFO] [ERROR] npm ERR! code ELIFECYCLE
[INFO] [ERROR] npm ERR! errno 1
[INFO] [ERROR] npm ERR! @adobe/cq-react-editable-components@1.3.2 postinstall: `node deprecated.js`
[INFO] [ERROR] npm ERR! Exit status 1
[INFO] [ERROR] npm ERR! 
[INFO] [ERROR] npm ERR! Failed at the @adobe/cq-react-editable-components@1.3.2 postinstall script.
[INFO] [ERROR] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[INFO] [ERROR] 
[INFO] [ERROR] npm ERR! A complete log of this run can be found in:
[INFO] [ERROR] npm ERR!     /home/circleci/.npm/_logs/2020-08-24T14_06_36_297Z-debug.log
```

looks like the renamed adobe npm module :

https://www.npmjs.com/package/@adobe/cq-react-editable-components

is causing the issue.

/cc @godanny86 @CezCz

--- a/src/main/archetype/ui.frontend.react/README.md
+++ b/src/main/archetype/ui.frontend.react/README.md
@@ -2,7 +2,7 @@
 
 This project was bootstrapped with [`create-react-app`](https://github.com/facebook/create-react-app).
 
-This application is built to consume the AEM model of a site. It will automatically generate the layout using the helper components from the [`@adobe/cq-react-editable-components`](https://www.npmjs.com/package/@adobe/cq-react-editable-components) package.
+This application is built to consume the AEM model of a site. It will automatically generate the layout using the helper components from the [`@adobe/aem-react-editable-components`](https://www.npmjs.com/package/@adobe/aem-react-editable-components) package.
 
 ${hash}${hash} Scripts
 

--- a/src/main/archetype/ui.frontend.react/package.json
+++ b/src/main/archetype/ui.frontend.react/package.json
@@ -10,9 +10,9 @@
     "sync": "aemsync -d -w ../ui.apps/src/main/content"
   },
   "dependencies": {
-    "@adobe/cq-react-editable-components": "^1.2.0",
-    "@adobe/cq-spa-component-mapping": "^1.0.3",
-    "@adobe/cq-spa-page-model-manager": "^1.1.0",
+    "@adobe/aem-react-editable-components": "^1.0.0",
+    "@adobe/aem-spa-component-mapping": "^1.0.0",
+    "@adobe/aem-spa-page-model-manager": "^1.0.0",
     "custom-event-polyfill": "^1.0.7",
     "dompurify": "^2.0.7",
     "history": "^4.10.1",

--- a/src/main/archetype/ui.frontend.react/src/App.js
+++ b/src/main/archetype/ui.frontend.react/src/App.js
@@ -1,4 +1,4 @@
-import { Page, withModel } from '@adobe/cq-react-editable-components';
+import { Page, withModel } from '@adobe/aem-react-editable-components';
 import React from 'react';
 
 // This component is the application entry point

--- a/src/main/archetype/ui.frontend.react/src/components/Page/Page.js
+++ b/src/main/archetype/ui.frontend.react/src/components/Page/Page.js
@@ -18,7 +18,7 @@ import {
   MapTo,
   Page,
   withComponentMappingContext
-} from '@adobe/cq-react-editable-components';
+} from '@adobe/aem-react-editable-components';
 import { withRoute } from '../RouteHelper/RouteHelper';
 
 require('./Page.css');

--- a/src/main/archetype/ui.frontend.react/src/components/Page/Page.test.js
+++ b/src/main/archetype/ui.frontend.react/src/components/Page/Page.test.js
@@ -14,7 +14,7 @@
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
-import { ModelManager } from '@adobe/cq-spa-page-model-manager';
+import { ModelManager } from '@adobe/aem-spa-page-model-manager';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import sinon from 'sinon';

--- a/src/main/archetype/ui.frontend.react/src/components/RouteHelper/RouteHelper.test.js
+++ b/src/main/archetype/ui.frontend.react/src/components/RouteHelper/RouteHelper.test.js
@@ -14,7 +14,7 @@
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
-import { ModelManager } from '@adobe/cq-spa-page-model-manager';
+import { ModelManager } from '@adobe/aem-spa-page-model-manager';
 import { configure, shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import React, { Component } from 'react';

--- a/src/main/archetype/ui.frontend.react/src/components/Text/Text.js
+++ b/src/main/archetype/ui.frontend.react/src/components/Text/Text.js
@@ -14,7 +14,7 @@
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
-import { MapTo } from '@adobe/cq-react-editable-components';
+import { MapTo } from '@adobe/aem-react-editable-components';
 import DOMPurify from 'dompurify';
 import React, { Component } from 'react';
 import extractModelId from '../../utils/extract-model-id';

--- a/src/main/archetype/ui.frontend.react/src/components/Text/Text.test.js
+++ b/src/main/archetype/ui.frontend.react/src/components/Text/Text.test.js
@@ -14,7 +14,7 @@
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
-import { ModelManager } from '@adobe/cq-spa-page-model-manager';
+import { ModelManager } from '@adobe/aem-spa-page-model-manager';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import sinon from 'sinon';

--- a/src/main/archetype/ui.frontend.react/src/index.js
+++ b/src/main/archetype/ui.frontend.react/src/index.js
@@ -21,7 +21,7 @@ import 'react-app-polyfill/stable';
 import 'react-app-polyfill/ie9';
 import 'custom-event-polyfill';
 
-import { Constants, ModelManager } from '@adobe/cq-spa-page-model-manager';
+import { Constants, ModelManager } from '@adobe/aem-spa-page-model-manager';
 import { createBrowserHistory } from 'history';
 import React from 'react';
 import { render } from 'react-dom';

